toxcore/tox_unpack.c

```diff
@@ -129,3 +129,75 @@ bool tox_unpack_user_status(Bin_Unpack *bu, Tox_User_Status *val)
     *val = tox_user_status_from_int(u32);
     return true;
 }
+
+bool tox_unpack_group_privacy_state(Bin_Unpack *bu, Tox_Group_Privacy_State *val)
+{
+    uint32_t u32;
+
+    if (!bin_unpack_u32(bu, &u32)) {
+        return false;
+    }
+
+    *val = (Tox_Group_Privacy_State)u32;
```

Comment:
This will error on ubsan when we fuzz it. You need the `_from_int` functions (that's why they exist).

Reply:
I missed that pr while rebasing ngc_events! (you can tell this looks like the others looked before)